test/functional/test.run.js

```diff
@@ -124,20 +124,9 @@ describe("jpm run", function () {
       var options = { cwd: paramDumpPath, env: { JPM_FIREFOX_BINARY: binary }};
       var proc = exec("run", options, function (err, stdout, stderr) {
         expect(err).to.not.be.ok;
-        expect(stdout).to.contain("console\.log: param-dump:")
+        expect(stdout).to.contain("console\.log: param-dump:");
         expect(stdout).to.contain("PARAMS DUMP START");
-        done();
-      });
-    });
-
-    it("logs out everything from console with -v", function (done) {
```

Comment:
It never worked correctly, if I'm not mistaken. We strip out all logs in testing and that `expect(stdout.split("\n").length).to.be.gt(2);` just applies to the two `Creating XPI` messages we forgot to log out with our own util.console. If you take that away it receives exactly the same output as the test above.